ooniapi/services/oonirun/src/oonirun/routers/v2.py

```diff
@@ -41,6 +41,7 @@ class OONIRunLinkNettest(BaseModel):
     inputs: List[str] = Field(
         default=[], title="list of input dictionaries for the nettest"
     )
+    # TODO(luis): Options and backend_options not in the new spec. Should be removed?
```

Comment:
We should for sure drop `backend_options`. I am not sure if it's useful to have a top level `options` that applies to all the inputs inside of the `nettest` or if it's redundant as we can include it directly inside of the `inputs_extra`.

@DecFox what do you think?

Reply:
So in thinking more about the topic of the `options` key, I don't think we can get rid of it. It's actually not redundant with `inputs_extra` since you would not be able to pass the options for a test using `inputs_extra` if that test takes no inputs.

As a result of this, I think we might have to keep `options`